src/frontend/js/widgets/UserLogin/index.tsx

```diff
@@ -73,10 +75,47 @@ const UserLogin = ({ profileUrls = {} }: UserLoginProps) => {
    */
   const { user, destroy, login, register } = useSession();
   const intl = useIntl();
+  const {
+    item: joanieUserProfile,
+    states: { isFetched: isJoanieUserProfileFetched },
+  } = useJoanieUserProfile();
+
+  const isAnonymous = user === null;
+  const isUserFetched = user !== undefined;
+  const isReady = isAnonymous || (isUserFetched && isJoanieUserProfileFetched);
```

Comment:
Do this could work instead? 

```suggestion
  const isReady = isAnonymous || !!joanieUserProfile;
```